packages/camera/camera/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.10.3+3
```

Comment:
Adding `// ignore`s is a dev-only change, so of the packages where the only ignoring deprecation should have the version change and changelog reverted.

Reply:
The CI will fail when you do that, which is fine; when I re-review I'll make sure it's only failing on the expected packages, then override that check.

Reply:
hey @tarrinneal Couldn't you override these analyzer checks? Since '//ignore' is a dev-only change. 

I did this and have to reverse